lib/index.js

```diff
@@ -0,0 +1,96 @@
+var bchLib = require('@owstack/satoshi-common-lib');
+
+// module information
+bchLib.version = 'v' + require('../package.json').version;
+
+bchLib.crypto.Signature.SIGHASH_FORKID = 0x40;
+
+var Constants = require('./common/constants');
+bchLib.Constants = Constants;
+
+bchLib.Networks.remove(bchLib.Networks.livenet);
```

Comment:
Should the base library even have a declared configuration? Should we make the base abstract?

Reply:
For now, the base still has the full test suite, prior to Segwit. I wanted to leave all of that in place for confidence. Can cleanup further in later steps.